bcda/models/repository.go

```diff
@@ -80,3 +81,8 @@ type jobRepository interface {
 type jobKeyRepository interface {
 	GetJobKeys(ctx context.Context, jobID uint) ([]*JobKey, error)
 }
+
+var (
```

Comment:
Can we move this to service.go since it's utilized there?

Reply:
Updated to move to `service.go`